src/github/github.controller.ts

```diff
@@ -25,7 +26,8 @@ export class GitHubController {
    * @returns number of commits of current month
    */
   @Get('commits')
-  @CacheTTL(36000)
+  @CacheTTL(36000000)
+  @UseInterceptors(CacheInterceptor)
```

Comment:
@rattrap With the latest version of dependencies if we specify the interceptor use in a class level you don't have a way to exclude an endpoint, so we're kinda forced to do this for now, hopefully they'll add an annotation later on.